src/js/claims-status/containers/AppealStatusPage.jsx

```diff
@@ -105,50 +106,52 @@ class AppealStatusPage extends React.Component {
     const { status } = appealStatusDescriptions(lastEvent, previousHistory);
 
     return (
-      <div className="claims-container">
-        <div className="row">
-          <div>
-            <h1 className="claims-container-title">Your Compensation Appeal Status</h1>
-            <p>This information is accurate as of {moment().format('MMMM DD, YYYY')}.</p>
+      <DowntimeNotification appName="appeals status tool" dependencies={[services.appeals]}>
```

Comment:
On this app, this is only going to show up for the downtime approaching case, correct? Because if appeals is down, they'll be stuck on the claim list page.

Reply:
It was implemented based purely off of the spreadsheet; I don't know enough about the application to know to do otherwise. Is this a special case? If it is we can hold off for later.

Reply:
Like I mentioned in another comment, this page is basically only accessed from the claims & appeals index page. If the appeals service is down, the appeals just won't show up in the index page, so the user won't be able to navigate here. Still, if they have it bookmarked or something, this'll help. What we do here will depend on what we do with the `ClaimsStatusApp`.